libgloss/arc/emsdp.specs

```diff
@@ -0,0 +1,19 @@
+
+%rename link emsdp_link
+%rename link_gcc_c_sequence emsdp_link_gcc_c_sequence
+%rename startfile emsdp_startfile
+
+*link:
+%(emsdp_link)
```

Comment:
Considering the binutils mod https://github.com/foss-for-synopsys-dwc-arc-processors/binutils-gdb/commit/e9dcba087e879c4e34a7400712901dc898fdf46f
You can just do:
```
%(emsdp_link) -marcv2elf --defsym=__TEXT_REGION_ORIGIN__=0x10000000 --defsym=__TEXT_REGION_LENGTH__=128k --defsym=__DATA_REGION_LENGTH__=128k
```